generator/osm2type.cpp

```diff
@@ -601,6 +601,7 @@ string DetermineSurface(OsmElement * p)
       "metal", "paved", "paving_stones", "sett", "unhewn_cobblestone", "wood"
   };
 
+  // All not explicitly listed surface types are considered unpaved good, e.g. "compacted", "fine_gravel".
```

Comment:
Is it intended that `surface=gibbering_gibberish_value` will be also "unpaved good"?

Reply:
IMO it makes sense to consider all "unknown" types as unpaved good, though I think it'll be better to have all common types listed explicitly.